subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -273,11 +275,26 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 	rx->link = link;
 	ftr->extra = NULL;
 
-	ftr->aux_w4next = 0;
+	ftr->aux_sched = 0U;
 
 	pdu = (void *)((struct node_rx_pdu *)rx)->pdu;
 	p = (void *)&pdu->adv_ext_ind;
 	if (!p->ext_hdr_len) {
+		data_len = pdu->len - PDU_AC_EXT_HEADER_SIZE_MIN;
```

Comment:
Is this right to assume that there is extended header even the length has value zero?